src/components/Header/Header.js

```diff
@@ -106,6 +107,7 @@ const Header = ({ links, isNightMode, setNightMode }) => {
 };
 
 const renderLink = (name, label, icon, classes) => {
+  const { t } = useTranslation();
```

Comment:
Line 110:17:  React Hook "useTranslation" is called in function "renderLink" which is neither a React function component or a custom React Hook function         react-hooks/rules-of-hooks